utils/anautilities.py

```diff
@@ -121,18 +121,32 @@ def getDirByAnaType(anaType, cName, ztrim=4):
     dataPath  = os.getenv('DATA_PATH')
 
     dirPath = ""
-    if anaType == "latency":
+    if anaType == "dacScanV3":
+        dirPath = "%s/%s"%(dataPath,anaType)
+    elif anaType == "latency":
         dirPath = "%s/%s/%s/trk/"%(dataPath,cName,anaType)
+    elif anaType == "sbitMonInt":
+        dirPath = "%s/%s/sbitMonitor/intTrig/"%(dataPath,cName)
+    elif anaType == "sbitMonRO":
+        dirPath = "%s/%s/sbitMonitor/readout/"%(dataPath,cName)
+    elif anaType == "sbitRatech":
+        dirPath = "%s/%s/sbitRate/perchannel/"%(dataPath,cName)
+    elif anaType == "sbitRateor":
+        dirPath = "%s/%s/sbitRate/channelOR/"%(dataPath,cName)
     elif anaType == "scurve":
         dirPath = "%s/%s/%s/"%(dataPath,cName,anaType)
+    elif anaType == "temperature":
+        dirPath = "%s/%s"%(dataPath,anaType)
```

Comment:
this monitoring isn't done per-chamber?

Reply:
The output TFile produced by dacScanV3.py may contain multiple chambers worth of information depending on the ohMask used at runtime.  If this design choice is undesirable than I will try to come up with a way to get one-to-one correspondence.

Reply:
Similarly for the temperature data

Reply:
OK